cluster/gce/config-test.sh

```diff
@@ -321,10 +321,11 @@ NODE_PROBLEM_DETECTOR_TAR_HASH="${NODE_PROBLEM_DETECTOR_TAR_HASH:-}"
 NODE_PROBLEM_DETECTOR_RELEASE_PATH="${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-}"
 NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS="${NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS:-}"
 
-CNI_STORAGE_PATH="${CNI_STORAGE_PATH:-https://storage.googleapis.com/kubernetes-release/network-plugins}"
-CNI_TAR_PREFIX="${CNI_TAR_PREFIX:-cni-plugins-amd64-}"
 CNI_VERSION="${CNI_VERSION:-}"
 CNI_SHA1="${CNI_SHA1:-}"
+# TODO(justaugustus); Post-0.7.5 CNI needs a "cni-plugins-linux-" prefix
+CNI_TAR_PREFIX="${CNI_TAR_PREFIX:-cni-plugins-amd64-}"
+CNI_GITHUB_DOWNLOAD_BASE_URL="${CNI_GITHUB_DOWNLOAD_BASE_URL:-https://github.com/containernetworking/plugins/releases/download}"
```

Comment:
Cloud we reuse CNI_STORAGE_PATH?
I think you can make the switch by setting the value of CNI_STORAGE_PATH and CNI_TAR_PREFIX

Reply:
@cadmuxe -- When I switched the links to retrieve from GitHub, `CNI_STORAGE_PATH` no longer made sense semantically as a variable name, so I changed it.

You'll see that now that the PR is using a different GCS location, instead of GitHub, I've reverted the variable name change.

Reply:
Thank you for making this PR.
/lgtm